Research/ResearchUI/iOS/RSDStepViewController.swift

```diff
@@ -739,6 +739,11 @@ open class RSDStepViewController : UIViewController, RSDStepController, RSDCance
             self.present(navVC, animated: true, completion: nil)
             return true
         }
+        else if let videoAction = action as? RSDVideoViewUIAction {
+            // For a videoView action, present an AVPlayerViewController modally.
+            RSDVideoViewController.present(action: videoAction, presenter: self)
```

Comment:
So... I personally prefer this pattern to the one above where the web view controller has a class method to instantiate the view controller. That said, I think I did this b/c at the time the "Apple preferred convention" was to have the presenter do the presenting and deferring that to the view controller being presented was a "no-no" (Don't remember why). Can you please check which is the *current* preferred pattern and either refactor the web view controller to use that or refactor this method to use the pattern above. Thanks!

Reply:
You're right, the presenter should do the presenting.  I'll switch it to that.

Reply:
Thanks for checking.